### PR TITLE
Adding datadog monitor "locked" option

### PR DESCRIPTION
**Pull request by tkensiski**
##### ISSUE TYPE
- Feature Pull Request
##### COMPONENT NAME

ansible-modules-extras/monitoring/datadog_monitor.py
##### ANSIBLE VERSION

```
ansible 2.1.1.0
  config file =
  configured module search path = Default w/o overrides
```
##### SUMMARY

Adding the "locked" option to the module. I need it for a project im working on so we can flag them as owned by our system or not flagged owned by the GUI.

Docs: http://docs.datadoghq.com/api/#monitor-create

Snippet from docs

```
options [optional]
A dictionary of options for the monitor. There are options that are common to all types as well as options that are specific to certain monitor types.
COMMON OPTIONS
...
locked a boolean indicating whether changes to to this monitor should be restricted to the creator or admins.
Default: False
...
```

### CODE REVIEW

**Comment by tkensiski** [comment]
Not sure what version file to bump. the main VERSION file in the root of this repo dosnt seem right but I may be wrong.

**Comment by tkensiski** [comment]
Im not sure where this version is that the ci system is complaining about. Once I know where that is Ill be more than happy to fix.

**Comment by gregdek** [comment]
Thanks @tkensiski. To the current maintainers, @skornehl please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by tkensiski** [comment]
@gregdek how do I fix the version issue that travis-ci is complaining about?

**Comment by gregdek** [comment]
Hi @tkensiski -- the problem is that whenever you add a new option, you have to document the version of Ansible for which the new option was added. 

Here's an example: https://github.com/ansible/ansible-modules-extras/blob/devel/cloud/amazon/ec2_remote_facts.py#L22

You don't need the separate VERSION file.

**Comment by gregdek** [comment]
Thanks @tkensiski for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by tkensiski** [comment]
@gregdek I tried to fix the version issue based on your example however it is still failing to pass tests.

**Comment by tkensiski** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @tkensiski. To the current maintainers, @skornehl please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by skornehl** [comment]
Thanks for this @tkensiski!

shipit

File: monitoring/datadog_monitor.py [hunk 1]
```diff
@@ -103,6 +103,11 @@
         description: ["A dictionary of thresholds by status. This option is only available for service checks and metric alerts. Because each of them can have multiple thresholds, we don't define them directly in the query."]
         required: false
         default: {'ok': 1, 'critical': 1, 'warning': 1}
+    locked:
+        description: ["A boolean indicating whether changes to to this monitor should be restricted to the creator or admins."]
```
Comment (resmo) [inline monitoring/datadog_monitor.py]: little typo spotted: to to

**Comment by gregdek** [comment]
Thanks again to @tkensiski for this PR, and thanks @skornehl for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by tkensiski** [comment]
Fixed the typo that @resmo found.

**Pull request merged**